reconcile/utils/metrics.py

```diff
@@ -2,13 +2,18 @@
 from prometheus_client import Histogram
 
 
-run_time = Gauge(name='qontract_reconcile_last_run_seconds',
-                 documentation='Last run duration in seconds',
-                 labelnames=['integration', 'shards', 'shard_id'])
+extra_labels = {'shard_key': None}
```

Comment:
@rporres it's set to None right here

Reply:
If I understand correctly, that is imported from `run-integegration.py` which is not in the manual execution path.